message/async/name.js

```diff
@@ -28,6 +28,8 @@ exports.create = function (api) {
         if (value.content.text.trim()) {
           return cb(null, titleFromMarkdown(value.content.text, 40) || fallbackName)
         }
+      } else if (value && typeof value.content.title === 'string') {
+        return cb(null, truncate(value.content.title, 40))
```

Comment:
nice one !

Reply:
maybe this should be above the `post` check, in the event people make posts which have titles in the future

Reply:
Yes good point!